### PR TITLE
feat(posts): add post about ApiaryUI - new version of Apiary Interactive Documentation

### PR DESCRIPTION
**Pull request by char0n**

### CODE REVIEW

File: _posts/2018-12-24-ApiaryUI.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+I would like to wish all Apiary users Mary Christmas and Happy new Year. We are thrilled to 
+welcome all Apiary users to try the new ApiaryUI - Interactive Documentation v4. 
+Follow this [help page](https://help.apiary.io/tools/interactive-documentation-v4/) to get information how to enable ApiaryUI on your existing API Projects.
+The help page also contains some additional information about features from Interactive Documentation v3
+that are currently missing in ApiaryUI.
+
+
+### Attributions
+
+I would like to give special thanks to Patrick Silva, Radek Podrazsky and Standa Opichal.
```
Comment (ajkl2533) [inline _posts/2018-12-24-ApiaryUI.md]:
```suggestion
 I would like to give special thanks to Patrick Silva, Radek Podrazky and Standa Opichal.
```

File: _posts/2018-12-24-ApiaryUI.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+the **ApiaryUI** - brand new, fully reviewed and refactored version of 
+[Apiary Interactive Documentation v3](https://help.apiary.io/tools/interactive-documentation/) (v3).
+
+ApiaryUI has already been deployed to [apiary.io](https://apiary.io/) on 20th of December 2018. 
+ApiaryUI is currently in what we call EAP (Early Access Program).
+Although it's already GA (Generally Available), it’s not yet fully production ready.
+We want to share it with the world now and get the invaluable feedback from you - Apiary users.
+
+ApiaryUI is implemented in a way, that allows complete separation from [apiary.io](https://apiary.io/)
+and possibly forming a new separate product in the future. Some features from v3 are still missing
```
Comment (kylef) [inline _posts/2018-12-24-ApiaryUI.md]:
```suggestion
 and possibly forming a new separate product in the future. Some features from our existing documentation product are still missing
```

I’m wondering if it makes sense to avoid the term “v3”, I am not sure that many of our customers or readers of this post will know what “v3” is exactly.

File: _posts/2018-12-24-ApiaryUI.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+ApiaryUI has full support for error containment and reporting. When the ApiaryUI intercept an unexpected failure
+it will contain it and display the Error Boundary view. This view contains button for downloading
+the complete Error Report. 
+
+
+### Next steps
+
+#### Speed
+
+We're currently parsing your Api Description Documentation on every browser view. This makes
```
Comment (kylef) [inline _posts/2018-12-24-ApiaryUI.md]:
```suggestion
We're currently parsing your API Description Documentation on every browser view. This makes
```

File: _posts/2018-12-24-ApiaryUI.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+[Apiary Interactive Documentation v3](https://help.apiary.io/tools/interactive-documentation/) 
+and [Embed](https://help.apiary.io/tools/embed/). 
+
+#### Development of missing features
+
+We'll be continuously delivering missing features from [Apiary Interactive Documentation v3](https://help.apiary.io/tools/interactive-documentation/) into ApiaryUI.
+To learn more about the missing features, please refer to [ApiaryUI help page](https://help.apiary.io/tools/interactive-documentation-v4/).
+
+
+I would like to wish all Apiary users Mary Christmas and Happy new Year. We are thrilled to
```
Comment (kylef) [inline _posts/2018-12-24-ApiaryUI.md]:
```suggestion
I would like to wish all Apiary users Merry Christmas and Happy new Year. We are thrilled to 
```

File: _posts/2018-12-24-ApiaryUI.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+### What's new in ApiaryUI
+
+ApiaryUI went through major product decisions. The concept of having three column layout stayed,
+but we reorganized the information displayed on each of them. Human Column is now designed
+to contain all the information needed for API Viewer to fully understand what the API is all about.
+Machine Columns sole responsibility is to display machine processing related information
+about one Http Transaction - Response/Request pair.
+
+Although ApiaryUI exclusively uses [API Elements](https://apielements.org/) as data format, it can also consume
+Api Description Document (ADD) formats like [API Blueprint](https://apiblueprint.org/) or [OpenAPI 2.0](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md) ([OpenAPI 3.0](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md) coming soon)
```
Comment (kylef) [inline _posts/2018-12-24-ApiaryUI.md]:
```suggestion
API Description Document (ADD) formats like [API Blueprint](https://apiblueprint.org/) or [OpenAPI 2.0](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md) ([OpenAPI 3.0](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md) coming soon)
```

File: _posts/2018-12-24-ApiaryUI.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+we'll them announce on [Apiary Blog](https://blog.apiary.io/).
+
+
+### What's new in ApiaryUI
+
+ApiaryUI went through major product decisions. The concept of having three column layout stayed,
+but we reorganized the information displayed on each of them. Human Column is now designed
+to contain all the information needed for API Viewer to fully understand what the API is all about.
+Machine Columns sole responsibility is to display machine processing related information
+about one Http Transaction - Response/Request pair.
```
Comment (kylef) [inline _posts/2018-12-24-ApiaryUI.md]:
```suggestion
about one HTTP Transaction - Response/Request pair.
```

File: _posts/2018-12-24-ApiaryUI.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+excerpt: Introduction version 4 of Apiary Interface Documentation
+layout: post
+date: 2018-12-24 12:00:00 +0200
+author: char0n
+published: true
+comments: true
+---
+
+After months of hard work and burning the midnight oil I am very proud to announce
+the **ApiaryUI** - brand new, fully reviewed and refactored version of
```
Comment (kylef) [inline _posts/2018-12-24-ApiaryUI.md]: I think “fully reviewed” is a little awkward in this sentence, although I understand your intent.

Perhaps re-imagined, or rebuilt from the ground up could work better.

File: _posts/2018-12-24-ApiaryUI.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+layout: post
+date: 2018-12-24 12:00:00 +0200
+author: char0n
+published: true
+comments: true
+---
+
+After months of hard work and burning the midnight oil I am very proud to announce
+the **ApiaryUI** - brand new, fully reviewed and refactored version of 
+[Apiary Interactive Documentation v3](https://help.apiary.io/tools/interactive-documentation/) (v3).
```
Comment (kylef) [inline _posts/2018-12-24-ApiaryUI.md]:
```suggestion
[our existing Interactive Documentation](https://help.apiary.io/tools/interactive-documentation/) (v3).
```

This sentence previously had a double “v3”, one inside the hyperlink and one just after in (brackets).

File: _posts/2018-12-24-ApiaryUI.md [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+
+ApiaryUI has already been deployed to [apiary.io](https://apiary.io/) on 20th of December 2018. 
+ApiaryUI is currently in what we call EAP (Early Access Program).
+Although it's already GA (Generally Available), it’s not yet fully production ready.
+We want to share it with the world now and get the invaluable feedback from you - Apiary users.
```
Comment (kylef) [inline _posts/2018-12-24-ApiaryUI.md]:
```suggestion
We want to share it with the world now and get invaluable feedback from you - Apiary users.
```

The use of “the” is unnecessary in this sentence.

File: _posts/2018-12-24-ApiaryUI.md [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+
+ApiaryUI is implemented in a way, that allows complete separation from [apiary.io](https://apiary.io/)
```
Comment (kylef) [inline _posts/2018-12-24-ApiaryUI.md]:
```suggestion
ApiaryUI is implemented in a way that allows complete separation from [apiary.io](https://apiary.io/)
```

This is already a small sentence and I don’t think it fits in to have a break there.

File: _posts/2018-12-24-ApiaryUI.md [hunk 11]
```diff
@@ -0,0 +1,131 @@
...
+and possibly forming a new separate product in the future. Some features from v3 are still missing
+in ApiaryUI. Some requested features not present in v3 were implemented to ApiaryUI.
+We’ve managed to fix tons of v3 bugs inside the new ApiaryUI.
```
Comment (kylef) [inline _posts/2018-12-24-ApiaryUI.md]:
```suggestion
We’ve managed to fix tonnes of existing bugs in our new ApiaryUI. 
```

This may be a controversial suggestion (tonnes vs tons) thought 😃.

File: _posts/2018-12-24-ApiaryUI.md [hunk 12]
```diff
@@ -0,0 +1,131 @@
...
+ApiaryUI is implemented in a way, that allows complete separation from [apiary.io](https://apiary.io/)
+and possibly forming a new separate product in the future. Some features from v3 are still missing
+in ApiaryUI. Some requested features not present in v3 were implemented to ApiaryUI.
+We’ve managed to fix tons of v3 bugs inside the new ApiaryUI. 
+
+As the architecture of ApiaryUI is very simple and solid, we are able to add new features
+and fix bugs in very rapid pace. Our extensive suite of unit, integration and e2e tests
+are helping with that. This can finally release further evolution of [API Blueprint](https://apiblueprint.org/) language.
+
+We are currently in a process of creating the deprecation strategies for
```
Comment (kylef) [inline _posts/2018-12-24-ApiaryUI.md]:
```suggestion
We are currently in a process of creating deprecation strategies for  
```

The place of “the” is a little awkward when we are using plural “strategies”

File: _posts/2018-12-24-ApiaryUI.md (hunk 7 above)
Comment (kylef) [inline _posts/2018-12-24-ApiaryUI.md]:
```suggestion
**ApiaryUI** - our brand new, fully rethought, and refactored version of 
```

I’d remove “the” in this construct.

**Comment by char0n** [comment]
@kylef thank you for seeing this. I did provided instruction here: https://github.com/apiaryio/apiaryio.github.com/pull/131/files#diff-2a3032127df61e3512bf94e9a96a8bd8R120

But if you missed it then I guess it's not clear enough from the text

File: _posts/2018-12-24-ApiaryUI.md (hunk 4 above)
Comment (honzajavorek) [reply _posts/2018-12-24-ApiaryUI.md]: The Christmas are over. In the Czech Republic we use Christmas to denote the whole week, but elsewhere they usually mean just the specific days when the festivities happen. I'd strip the Christmas from the wish and focus on the new year 😄

File: _posts/2018-12-24-ApiaryUI.md [hunk 13]
```diff
@@ -0,0 +1,131 @@
...
+
+#### Development of missing features
+
+We'll be continuously delivering missing features from [Apiary Interactive Documentation v3](https://help.apiary.io/tools/interactive-documentation/) into ApiaryUI.
+To learn more about the missing features, please refer to [ApiaryUI help page](https://help.apiary.io/tools/interactive-documentation-v4/).
+
+
+I would like to wish all Apiary users Mary Christmas and Happy new Year. We are thrilled to 
+welcome all Apiary users to try the new ApiaryUI - Interactive Documentation v4. 
+Follow this [help page](https://help.apiary.io/tools/interactive-documentation-v4/) to get information how to enable ApiaryUI on your existing API Projects.
```
Comment (honzajavorek) [inline _posts/2018-12-24-ApiaryUI.md]:
```suggestion
Follow [this tutorial in Apiary Help](https://help.apiary.io/tools/interactive-documentation-v4/) to learn about how you can start using ApiaryUI on your Apiary projects today.
```

File: _posts/2018-12-24-ApiaryUI.md [hunk 14]
```diff
@@ -0,0 +1,131 @@
...
+The help page also contains some additional information about features from Interactive Documentation v3
```
Comment (honzajavorek) [inline _posts/2018-12-24-ApiaryUI.md]: This is repeating what has been already written a few lines above.

File: _posts/2018-12-24-ApiaryUI.md (hunk 1 above)
Comment (honzajavorek) [reply _posts/2018-12-24-ApiaryUI.md]: @ajkl2533 Your suggestion adds an extraneous space in the beginning of the sentence. Otherwise it seems legit 😄

File: _posts/2018-12-24-ApiaryUI.md [hunk 15]
```diff
@@ -0,0 +1,131 @@
...
+that are currently missing in ApiaryUI.
+
+
+### Attributions
+
+I would like to give special thanks to Patrick Silva, Radek Podrazsky and Standa Opichal.
+These guys formed a core of new Documentation Team, did an exceptional job and make the delivery of ApiaryUI possible.
```
[the remark this answers is not in the document]
Comment (honzajavorek) [reply _posts/2018-12-24-ApiaryUI.md]: I'd avoid the word guys in this context: why not 'They formed' ?

File: _posts/2018-12-24-ApiaryUI.md [hunk 16]
```diff
@@ -0,0 +1,131 @@
...
+
+Thank you guys and keep up the good work!
```
Comment (honzajavorek) [inline _posts/2018-12-24-ApiaryUI.md]:
```suggestion
Thank you and keep up the good work!
```

Comment (honzajavorek) [reply _posts/2018-12-24-ApiaryUI.md]: 'guys' seems redundant again. For some it may sound more like 'dudes', I'd better skip that word or choose different wording.

File: _posts/2018-12-24-ApiaryUI.md [hunk 17]
```diff
@@ -0,0 +1,131 @@
...
+ApiaryUI is compatible with screen readers and even specially-abled people should navigate in it with no problems.
+Full support for touch events is guaranteed.
+
+##### Keyboard navigation
+
+ApiaryUI supports keyboard navigation for people who cannot use a mouse or other pointing device, 
+or who want to use a keyboard as much as possible.
+
+
+#### Mobile First (building for the future)
```
[the remark this answers is not in the document]
Comment (char0n) [reply _posts/2018-12-24-ApiaryUI.md]: I've stolen the `Mobile First` from https://foundation.zurb.com/mobile-first.html. Not sure how it should be correctly.

File: _posts/2018-12-24-ApiaryUI.md [hunk 18]
```diff
@@ -0,0 +1,131 @@
+---
+title: ApiaryUI - Interactive Documentation v4
```
[the remark this answers is not in the document]
Comment (char0n) [reply _posts/2018-12-24-ApiaryUI.md]: Removed

File: _posts/2018-12-24-ApiaryUI.md [hunk 19]
```diff
@@ -0,0 +1,131 @@
...
+excerpt: Introduction version 4 of Apiary Interface Documentation
```
[the remark this answers is not in the document]
Comment (char0n) [reply _posts/2018-12-24-ApiaryUI.md]: removed

File: _posts/2018-12-24-ApiaryUI.md [hunk 20]
```diff
@@ -0,0 +1,131 @@
...
+We are currently in a process of creating the deprecation strategies for  
+[Apiary Interactive Documentation v3](https://help.apiary.io/tools/interactive-documentation/)
+and [Embed](https://help.apiary.io/tools/embed/). When we have those strategies fully formed
+we'll them announce on [Apiary Blog](https://blog.apiary.io/).
+
+
+### What's new in ApiaryUI
+
+ApiaryUI went through major product decisions. The concept of having three column layout stayed,
+but we reorganized the information displayed on each of them. Human Column is now designed
```
[the remark this answers is not in the document]
Comment (char0n) [reply _posts/2018-12-24-ApiaryUI.md]: Used explanation from v3 help pages.

File: _posts/2018-12-24-ApiaryUI.md (hunk 7 above)
Comment (char0n) [reply _posts/2018-12-24-ApiaryUI.md]: used: `rethought`

File: _posts/2018-12-24-ApiaryUI.md [hunk 21]
```diff
@@ -0,0 +1,131 @@
...
+to contain all the information needed for API Viewer to fully understand what the API is all about.
+Machine Columns sole responsibility is to display machine processing related information
```
[the remark this answers is not in the document]
Comment (char0n) [reply _posts/2018-12-24-ApiaryUI.md]: Provided explanation and fixed the apostrophe

File: _posts/2018-12-24-ApiaryUI.md (hunk 17 above)
[the remark this answers is not in the document]
Comment (char0n) [reply _posts/2018-12-24-ApiaryUI.md]: Used `Mobile first`

File: _posts/2018-12-24-ApiaryUI.md [hunk 22]
```diff
@@ -0,0 +1,131 @@
...
+displaying of your API Projects via ApiaryUI unnecessarily slower. We plan to cache the result
+of the parsing and skip the parsing phase of ApiaryUI. This should rapidly increase the
+rendering times.  
+
+
+#### Deprecation strategies
+
+As I mentioned earlier, we are in the process of forming deprecation strategies for 
+[Apiary Interactive Documentation v3](https://help.apiary.io/tools/interactive-documentation/) 
+and [Embed](https://help.apiary.io/tools/embed/).
```
[the remark this answers is not in the document]
Comment (char0n) [reply _posts/2018-12-24-ApiaryUI.md]: Deleted from the begging and moved to appropriate section

File: _posts/2018-12-24-ApiaryUI.md (hunk 18 above)
Comment (Almad) [inline _posts/2018-12-24-ApiaryUI.md]: I'd refrain from using v4...v3 is here for so long noone remembers anything before that, imho. So I think something like ApiaryUI—rethought for speed or something like that would fit better.

File: _posts/2018-12-24-ApiaryUI.md (hunk 7 above)
Comment (Almad) [reply _posts/2018-12-24-ApiaryUI.md]: Agreed. I think in the fist paragraph, we should focus on what does it bring for user, and in that sense, first thing on my mind is speed.

File: _posts/2018-12-24-ApiaryUI.md [hunk 23]
```diff
@@ -0,0 +1,131 @@
...
+author: char0n
+published: true
+comments: true
+---
+
+After months of hard work and burning the midnight oil I am very proud to announce
+the **ApiaryUI** - brand new, fully reviewed and refactored version of 
+[Apiary Interactive Documentation v3](https://help.apiary.io/tools/interactive-documentation/) (v3).
+
+ApiaryUI has already been deployed to [apiary.io](https://apiary.io/) on 20th of December 2018.
```
Comment (Almad) [inline _posts/2018-12-24-ApiaryUI.md]: I'd be playful, how about something like "We already gave ApiaryUI to our betatesters as a Christmas gift 🎁 "?

File: _posts/2018-12-24-ApiaryUI.md [hunk 24]
```diff
@@ -0,0 +1,131 @@
...
+published: true
+comments: true
+---
+
+After months of hard work and burning the midnight oil I am very proud to announce
+the **ApiaryUI** - brand new, fully reviewed and refactored version of 
+[Apiary Interactive Documentation v3](https://help.apiary.io/tools/interactive-documentation/) (v3).
+
+ApiaryUI has already been deployed to [apiary.io](https://apiary.io/) on 20th of December 2018. 
+ApiaryUI is currently in what we call EAP (Early Access Program).
```
Comment (Almad) [inline _posts/2018-12-24-ApiaryUI.md]: EAP and GA totally sounds like corporate newspeak. Can't we rephrase into something akin to "We are making it available for everyone, but before it will become THE API documentation, we want to hear from you"...>?

Comment (Almad) [reply _posts/2018-12-24-ApiaryUI.md]: If EAP and GA is mandated by someone, I'd turn the explanations the other way around and put shortcuts into the parenthesis, not vice versa.

File: _posts/2018-12-24-ApiaryUI.md (hunk 2 above)
Comment (Almad) [reply _posts/2018-12-24-ApiaryUI.md]: "Some features" spreads FUD. I'd specify which at the end of article and say whether we do plan to implement them or not.

File: _posts/2018-12-24-ApiaryUI.md [hunk 25]
```diff
@@ -0,0 +1,131 @@
...
+Although it's already GA (Generally Available), it’s not yet fully production ready.
+We want to share it with the world now and get the invaluable feedback from you - Apiary users.
+
+ApiaryUI is implemented in a way, that allows complete separation from [apiary.io](https://apiary.io/)
+and possibly forming a new separate product in the future. Some features from v3 are still missing
+in ApiaryUI. Some requested features not present in v3 were implemented to ApiaryUI.
```
Comment (Almad) [inline _posts/2018-12-24-ApiaryUI.md]: Same here, "those features everyone asked for" (list) :)

File: _posts/2018-12-24-ApiaryUI.md [hunk 26]
```diff
@@ -0,0 +1,131 @@
...
+We’ve managed to fix tons of v3 bugs inside the new ApiaryUI. 
+
+As the architecture of ApiaryUI is very simple and solid, we are able to add new features
+and fix bugs in very rapid pace. Our extensive suite of unit, integration and e2e tests
```
Comment (Almad) [inline _posts/2018-12-24-ApiaryUI.md]: "in very" -> "at a very", or not @kylef ?

File: _posts/2018-12-24-ApiaryUI.md [hunk 27]
```diff
@@ -0,0 +1,131 @@
...
+are helping with that. This can finally release further evolution of [API Blueprint](https://apiblueprint.org/) language.
```
Comment (Almad) [inline _posts/2018-12-24-ApiaryUI.md]: I wouldn't disclose the last sentence. While true, our customers are not aware and this makes us either a laughing stock and/or a very particular promise. I wouldn't promise, I'd deliver. 

Let's start the APIB feature announcement blog post with "Thanks to our new documentation, we've been able to release...".

File: _posts/2018-12-24-ApiaryUI.md [hunk 28]
```diff
@@ -0,0 +1,131 @@
...
+
+We are currently in a process of creating the deprecation strategies for  
+[Apiary Interactive Documentation v3](https://help.apiary.io/tools/interactive-documentation/)
+and [Embed](https://help.apiary.io/tools/embed/). When we have those strategies fully formed
+we'll them announce on [Apiary Blog](https://blog.apiary.io/).
```
Comment (Almad) [inline _posts/2018-12-24-ApiaryUI.md]: I'd opt for "we'll announce them here".

File: _posts/2018-12-24-ApiaryUI.md (hunk 12 above)
Comment (Almad) [reply _posts/2018-12-24-ApiaryUI.md]: Sounds formal to me. I'd go with something like "This version will replace our current version in the future. The exact timing and strategy is still to be determined and is waiting for your feedback".

File: _posts/2018-12-24-ApiaryUI.md [hunk 29]
```diff
@@ -0,0 +1,131 @@
...
+
+### What's new in ApiaryUI
+
+ApiaryUI went through major product decisions. The concept of having three column layout stayed,
+but we reorganized the information displayed on each of them. Human Column is now designed
+to contain all the information needed for API Viewer to fully understand what the API is all about.
+Machine Columns sole responsibility is to display machine processing related information
+about one Http Transaction - Response/Request pair.
+
+Although ApiaryUI exclusively uses [API Elements](https://apielements.org/) as data format, it can also consume
```
Comment (Almad) [inline _posts/2018-12-24-ApiaryUI.md]: as _a_ data format, I think

File: _posts/2018-12-24-ApiaryUI.md [hunk 30]
```diff
@@ -0,0 +1,131 @@
...
+Api Description Document (ADD) formats like [API Blueprint](https://apiblueprint.org/) or [OpenAPI 2.0](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md) ([OpenAPI 3.0](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md) coming soon)
+and it parses the ADDs in Web Worker inside the browser. This means that ApiaryUI is completely
+backend-less and does not need the server as it's backend.
```
Comment (Almad) [inline _posts/2018-12-24-ApiaryUI.md]: Redundant: either "does not need server" (preferred) or "is completely backend-less".

File: _posts/2018-12-24-ApiaryUI.md [hunk 31]
```diff
@@ -0,0 +1,131 @@
...
+- Java
+
+#### Print support
+
+ApiaryUI is fully print friendly. You can easily print the rendered documentation directly on paper or into PDF.
+
+
+#### Accessibility
+
+ApiaryUI complies with latest accessibility standards. We care about correctness of HTML semantics and proper use of Aria attributes.
```
Comment (Almad) [inline _posts/2018-12-24-ApiaryUI.md]: I'd link Aria

File: _posts/2018-12-24-ApiaryUI.md [hunk 32]
```diff
@@ -0,0 +1,131 @@
...
+ApiaryUI is compatible with screen readers and even specially-abled people should navigate in it with no problems.
```
Comment (Almad) [inline _posts/2018-12-24-ApiaryUI.md]: TIL specially-abled, although [some people do not like it](https://adventuresbeyondbarriers.com/tag/specially-abled/).

File: _posts/2018-12-24-ApiaryUI.md [hunk 33]
```diff
@@ -0,0 +1,131 @@
...
+Mobile devices are already eclipsing desktops in adoption and internet use — that's why ApiaryUI was built from
+the ground up to support any kind of device, any size screen, with any resolution. 
+ApiaryUI behaves and looks differently on mobile and desktop resolutions to maximize the user experience. 
+
+#### Error Boundaries
+
+ApiaryUI has full support for error containment and reporting. When the ApiaryUI intercept an unexpected failure
+it will contain it and display the Error Boundary view. This view contains button for downloading
+the complete Error Report. 
+
```
Comment (Almad) [inline _posts/2018-12-24-ApiaryUI.md]: I think this deserves further explanation on why you should care. As in "this means that if there is any problem, you are still in control of your data, but you may decide to share it with us—and we'll have your complete state, which will allow us to fix bugs much faster".

File: _posts/2018-12-24-ApiaryUI.md [hunk 34]
```diff
@@ -0,0 +1,131 @@
...
+
+### Next steps
+
+#### Speed
+
+We're currently parsing your Api Description Documentation on every browser view. This makes
+displaying of your API Projects via ApiaryUI unnecessarily slower. We plan to cache the result
```
Comment (Almad) [inline _posts/2018-12-24-ApiaryUI.md]: Confident delivery style. "We plan to" -> "We'll"

File: _posts/2018-12-24-ApiaryUI.md (hunk 1 above)
Comment (Almad) [reply _posts/2018-12-24-ApiaryUI.md]: If people are OK with it, I'd make the names link, to put some human face to it. To not make it a personal promo, I'd to Github / Twitter / Linkedin, in that order of preference.

File: _posts/2018-12-24-ApiaryUI.md (hunk 16 above)
Comment (Almad) [inline _posts/2018-12-24-ApiaryUI.md]: While I like and fully support the attribution of the team, ending the article with it makes this a bit self-serving and worth an internal announcement and/or "how v4 was baked" blog post (which I would totally love to read and if you are up for it, I think it's totally worth writing!), but not a product announcement. 

I think the article should end with CTA for trying the documentation out and with thank you to the users since they are the audience of the article.

I also think both thank yous are possible within the same article, but it requires some thought and reshuffling of the content.

File: _posts/2018-12-24-ApiaryUI.md (hunk 32 above)
Comment (patricksmms) [reply _posts/2018-12-24-ApiaryUI.md]: Indeed. Even our CEO refers to these users as _disabled_  in http://www.oracle.com/us/corporate/accessibility/index.html.

BTW accessibility isn't only meant for disabled users, it also serves power users.

I would rephrase to something like:

> ApiaryUI is supports the usage of screen readers and keyboard navigation. Whether you have a disability, or a personal usage preference, it allows you to choose how to interact with the interface.

Comment (patricksmms) [reply _posts/2018-12-24-ApiaryUI.md]: And probably I would drop `ApiaryUI complies with latest accessibility standards`. We don't comply yet with minimum contrast ratio and some other nuances of the latest standards.

File: _posts/2018-12-24-ApiaryUI.md (hunk 26 above)
Comment (kylef) [reply _posts/2018-12-24-ApiaryUI.md]: TBH I'd just drop `very`:

```diff
- we are able to add new features and fix bugs in very rapid pace
+ we are able to add new features and fix bugs at a rapid pace.
```

File: _posts/2018-12-24-ApiaryUI.md (hunk 34 above)
Comment (char0n) [reply _posts/2018-12-24-ApiaryUI.md]: Changed

File: _posts/2018-12-24-ApiaryUI.md (hunk 33 above)
Comment (char0n) [reply _posts/2018-12-24-ApiaryUI.md]: Changed. Thanks

File: _posts/2018-12-24-ApiaryUI.md (hunk 32 above)
Comment (char0n) [reply _posts/2018-12-24-ApiaryUI.md]: Used disabled, dropped some sentences and rephrase the thing.

File: _posts/2018-12-24-ApiaryUI.md (hunk 31 above)
Comment (char0n) [reply _posts/2018-12-24-ApiaryUI.md]: Linked

File: _posts/2018-12-24-ApiaryUI.md (hunk 30 above)
Comment (char0n) [reply _posts/2018-12-24-ApiaryUI.md]: Already removed in previous comments. `is completely
backend-less` used.

File: _posts/2018-12-24-ApiaryUI.md (hunk 27 above)
Comment (char0n) [reply _posts/2018-12-24-ApiaryUI.md]: Removed

File: _posts/2018-12-24-ApiaryUI.md (hunk 23 above)
Comment (char0n) [reply _posts/2018-12-24-ApiaryUI.md]: Used

File: _posts/2018-12-24-ApiaryUI.md (hunk 18 above)
Comment (char0n) [reply _posts/2018-12-24-ApiaryUI.md]: used `New Interactive Documentation`. Dropped v3, v4 references

**Comment by char0n** [comment]
Everything addressed. Waiting for another round ;]

File: _posts/2018-12-24-ApiaryUI.md [hunk 35]
```diff
@@ -115,7 +115,7 @@ We'll be continuously delivering missing features from [Apiary Interactive Docum
 To learn more about the missing features, please refer to [ApiaryUI help page](https://help.apiary.io/tools/interactive-documentation-v4/).
 
 
-I would like to wish all Apiary users Mary Christmas and Happy new Year. We are thrilled to 
+I would like to wish all Apiary users Merry Christmas and Happy new Year. We are thrilled to 
 welcome all Apiary users to try the new ApiaryUI - Interactive Documentation v4. 
 Follow this [help page](https://help.apiary.io/tools/interactive-documentation-v4/) to get information how to enable ApiaryUI on your existing API Projects.
```
Comment (paraskakis) [inline _posts/2018-12-24-ApiaryUI.md]: Maybe just Happy New Year?
